AES.ApplicationSvc.Tests/ApplicationSvcUnitTests.cs

```diff
@@ -472,5 +481,512 @@ private ApplicationInfoContract PartialApp4_BadAnswers()
 
             return app;
         }
+
+        [TestMethod]
+        public void TC_GetApplicantsAwaitingScreening()
+        {
+            clearDb();
+
+            var applicationService = new ApplicationSvc();
+
+            SetupValidWaitingCallApp();
+
+            var applicantsAwaitingCalls = applicationService.GetApplicantsAwaitingCalls(new TimeSpan(6, 30, 0));
```

Comment:
So, I know what we need is a TimeSpan, but this is an API. The user doesn't need to know how we're working with the Data, and when they call it, it makes much more sense to be able to call it with DateTime.Now. Let's change the API to use DateTime, and convert to a TimeSpan inside the method that is called.